content/docs/services/cloud-gov-service-account.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: cloud.gov service account
 name: "cloud-gov-service-account"
-description: "Manage cloud.gov service accounts with access to your organization"
+description: "cloud.gov service accounts with access to your organization"
```

Comment:
Yeah, I struggled with this because we only have a single plan, and it's kind of open ended what these could be used for, or what plans might appear in the future. I like your suggestion.